lib/generators/rest_api_generator/resource_generator.rb

```diff
@@ -40,5 +40,15 @@ def build_model_attributes
       end
       model_attributes
     end
+
+    def define_scope
+      if options["scope"].empty?
+        API_CONTROLLER_DIR_PATH
+      else
+        parts = options["scope"].split(".")
+        API_CONTROLLER_DIR_PATH + "/#{parts[0]}/#{parts[1]}"
```

Comment:
Same thing from version method, can we change to work with n scopes?

Reply:
done, now it works with n versions